### PR TITLE
Add support for bootstrap-style alert boxes in notebooks

### LINKED ISSUE #8399: Adding pre-defined HTML boxes as in Jupyter Notebook.
Would it be possible to add some pre-defined CSS styling for HTML boxes, as it already happens in Jupyter Notebook?
(see https://gist.github.com/DanielKotik/4b81480c479a57e0dd13ac4d153e4451 for reference).

Now I add the following `style` to every Markdown cell I write to have them available as a workaround, but it would be nice to have them out-of-the-box.

```html
<style>
tt {
    font-family:
    "MonoLisa",
    "Courier Prime",
    "Courier New",
    "Inconsolata",
    "Lucida Console",
    "Menlo",
    "Monaco", 
    monospace;
    color: "black";
    font: 10px;
}

alert {
    display: inline-block;
}
    
.alert-block{
    margin: 20px;
    width: 92%;
    padding: 15px;
    border-radius: 4px;
    font: 16px Tahoma, Alegreya, Garamond, Times;
    font-weight: 400;
}

.alert-info {
    color: #0399A9;
    background: #B2EBF3;
    border: 2px solid #4FD1E1;
}

.alert-warning {
    color: #F67C00;
    background: #FFE0B2;
    border: 2px solid #FFC165;
}

.alert-danger {
    color: #C4453E;
    background: #F7CFD3;
    border: 2px solid #E29C9B;
}

.alert-success {
    color: #388E3C;
    background: #C8E6C9;
    border: 2px solid #81C784;
}
</style>
```

### PR DESCRIPTION
**Pull request by eric-wieser**
This PR fixes microsoft/vscode-jupyter#8399

### CODE REVIEW

File: extensions/markdown-language-features/notebook/index.ts [hunk 1]
```diff
@@ -7,6 +7,9 @@ import * as DOMPurify from 'dompurify';
 import MarkdownIt from 'markdown-it';
 import type * as MarkdownItToken from 'markdown-it/lib/token';
 import type { ActivationFunction } from 'vscode-notebook-renderer';
+import { editorErrorForeground, editorWarningForeground, editorInfoForeground,
```
Comment (mjbvz) [inline extensions/markdown-language-features/notebook/index.ts:10]: This is an extension so you can't import from `vs` 

All of these colors are instead already exposed as css variables

**Comment by mjbvz** [comment]
IMO this is too specific to belong in core VS Code. It should come from an extension, either a custom one or jupyter

File: extensions/markdown-language-features/notebook/index.ts (hunk 1 above)
Comment (eric-wieser) [reply extensions/markdown-language-features/notebook/index.ts:10]: I thought something like that might be the case. How can I find out what the CSS variable names are?

**Comment by mjbvz** [comment]
`blockquote` is a built-in html element. Bootstrap is just a 3d party framework

**Comment by eric-wieser** [comment]
Sure, but the bootstrap `alert` styles seems to have become a defacto feature of jupyter notebooks: They're supported in:

* Jupyter's own web renderer
* Github's web renderer; see this notebook for instance:
https://github.com/sigproc/cued_sf2_lab/blob/6a63558466eb147a640f5b19ab5e7ce3621833f4/1-4-intro-and-familiarisation.ipynb
* `nbsphinx`'s renderer: https://nbsphinx.readthedocs.io/en/0.2.7/markdown-cells.html#Info/Warning-Boxes

**Comment by mjbvz** [comment]
It sounds like it may belong in Jupyter then. This code or core VS Code applies to every kind of notebook. Bootstrap specific stuff does not belong here

**Comment by eric-wieser** [comment]
> This code or core VS Code applies to every kind of notebook.

I think this sounds like the source of my confusion; what other types of notebook are there?

**Comment by eric-wieser** [comment]
Anyway, thanks for the review! Would you be able to advise on https://github.com/microsoft/vscode/pull/149368#discussion_r871638930 so that when I PR this downstream to the jupyter plugin I don't make the same mistake?

**Comment by mjbvz** [comment]
You can use the webview css variables for this: https://code.visualstudio.com/api/extension-guides/webview#theming-webview-content

**Pull request closed without merging**